doc/proposals/scorecard-user-tests.md

```diff
@@ -0,0 +1,154 @@
+# User Defined Tests for the Operator Scorecard
```

Comment:
Should we update title, background, and goals to be more plugin-centric rather than focusing on the user-defined tests? 

It's definitely still relevant to this proposal. I think user-defined tests and increased flexibility for test setup/teardown are the motivations for the plugin system, not necessarily the direct goal.